discordSuperUtils/imaging.py

```diff
@@ -2,6 +2,8 @@
 
 import os
 import textwrap
+import time
+import requests
```

Comment:
Please, do not use requests in asynschournus programming! requests is a blocking library, and will block the main thread, meaning, while the request doesn't have a response, any commands, events, etc, won't be called!